doc: update the v3.0 migration guide (#24344)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/migration/migration_guide_3.0.rst

      Using Bluetooth LE legacy pairing introduces, among others, a risk of passive eavesdropping.
      Supporting Bluetooth LE legacy pairing makes devices vulnerable to downgrade attacks.

CRACEN initialization
---------------------

.. toggle::

   In the |NCS| versions 2.8.0 and 2.9.0, you need to explicitly configure the CRACEN initialization.

>    This was done by adding the :kconfig:option:`CONFIG_CRACEN_LOAD_MICROCODE` Kconfig option to the MCUboot configuration, where you could specify when CRACEN should be initialized.

Reviewer: ```suggestion
   This was done by adding the :kconfig:option:`CONFIG_CRACEN_LOAD_MICROCODE` Kconfig option to the image configuration, which selects given image to initialize CRACEN.
```
